bin/run-lerna.js

```diff
@@ -31,9 +32,5 @@ async function run(argv, options) {
 }
 
 module.exports = run;
-if (require.main === module) {
-  run(process.argv).catch(err => {
-    console.error(err);
-    process.exit(1);
-  });
-}
+
+main(module, run, process.argv);
```

Comment:
Let's pick a more descriptive name please, one that will make it more clear what is this line going to do.

For example:

```suggestion
runIfMainCliModule(module, run, process.argv);
```

Reply:
I renamed to `runMain()`